lib/ansible/modules/system/semodule.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
+# Copyright: (c) 2019, Philip Bove <phil@bove.online>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: semodule
+short_description: Manage selinux policy modules
+description:
+  - Use this module to manage selinux policy modules from Type Enforcement files
+version_added: "2.9"
```

Comment:
Hi @bandit145, You need to bump this to 2.10 now.

Reply:
Will do.